crates/goose/src/session/session_manager.rs

```diff
@@ -166,10 +176,10 @@ impl SessionManager {
             .map(Arc::clone)
     }
 
-    pub async fn create_session(working_dir: PathBuf, description: String) -> Result<Session> {
+    pub async fn create_session(working_dir: PathBuf, name: Option<String>) -> Result<Session> {
```

Comment:
why do we want to make the session name optional? I could see the case for dropping this completely and just either wait for the system or the user to set it